sources/packages/backend/libs/integrations/src/cas/models/cas-service.model.ts

```diff
@@ -47,6 +49,32 @@ export class CASAuthDetails {
   expires_in: number;
 }
 
+/**
+ * Default amount of time to a token be expired and be
+ * considered candidate to be renewed.
+ */
+const CAS_TOKEN_RENEWAL_SECONDS = 60;
+
+/**
+ * Cache the CAS token to be reused.
+ */
+export class CachedCASAuthDetails {
+  private readonly renewalTime: Date;
+  constructor(public readonly authDetails: CASAuthDetails) {
```

Comment:
public not required to be explicitly mentioned.

Reply:
Removed.